src/ansys/fluent/core/services/meshing_queries.py

```diff
@@ -642,85 +658,170 @@ def GetCellZoneAtLocation(self, location) -> Any:
         return response.cell_zone_id
 
     def GetZonesOfType(self, type) -> Any:
-        """GetZonesOfType."""
+        """
+        Return a list of zones of the specified default zone type.
+
+        .. code-block:: python
+
+            >>> meshing_session.meshing_queries.GetZonesOfType("velocity-inlet")
```

Comment:
I feel that we have inherited an interface that's quite difficult to use and that we'll want to improve it as muchas we can at the Python level. We can have a discussion about that.

Reply:
Sure.

Reply:
@seanpearsonuk 

We have improved error handling by introducing a function to give allowed arguments. 

Here is an example - 

![image](https://github.com/pyansys/pyfluent/assets/106588300/efaee526-2a2b-46b8-ac9d-1dcf05c7785c)

`--------------------------------------------`

![image](https://github.com/pyansys/pyfluent/assets/106588300/3e6162b7-8d42-489b-8b34-a59eb1e67d4d)

Reply:
![image](https://github.com/pyansys/pyfluent/assets/106588300/0aa9b101-996b-4651-be73-a3ca35c1520a)

Reply:
![image](https://github.com/pyansys/pyfluent/assets/106588300/81caae5f-e91f-462d-adef-6d2a209464bf)

Reply:
![image](https://github.com/pyansys/pyfluent/assets/106588300/6dd057f5-60b9-477a-8460-b51d646f85be)